schema/pgsql/schema.sql

```diff
@@ -21,6 +21,55 @@ CREATE TYPE comment_type AS ENUM ( 'comment', 'ack' );
 CREATE TYPE notification_type AS ENUM ( 'downtime_start', 'downtime_end', 'downtime_removed', 'custom', 'acknowledgement', 'problem', 'recovery', 'flapping_start', 'flapping_end' );
 CREATE TYPE history_type AS ENUM ( 'notification', 'state_change', 'downtime_start', 'downtime_end', 'comment_add', 'comment_remove', 'flapping_start', 'flapping_end', 'ack_set', 'ack_clear' );
 
+/* For the search bar suggestions, Icinga DB Web always issues SQL queries with text values,
+   even if the value should be a number, such as for check_interval.
+   This fails for PostgreSQL as it cannot match numbers to text. To support this,
```

Comment:
This is misleading. DB Web doesn't just pass numbers as text, it's rather passing numbers encapsulated by wildcards (e.g. `%4%`) which is only possible by passing text. The solution in DB Web also is not by offering different operators but by disallowing wildcards for integer columns.